README.md

```diff
@@ -1285,6 +1286,24 @@ Then install the new AWX Operator by following the instructions in [Basic Instal
 
 Once the new AWX Operator is up and running, your AWX deployment will also be upgraded.
 
+#### Disable IPV6 for install and upgrade
```

Comment:
Could we change the name of this section to just `Disable IPv6`?

Reply:
@rooftopcellist  Updated the requested changes it makes sense.Only thing is unable to make disable ipv6 section highlighted like other sections.